generic/pg-goals.el

```diff
@@ -79,6 +79,63 @@ May enable proof-by-pointing or similar features.
 ;;
 ;; Goals buffer processing
 ;;
+
+;; update *goals* buffer modeline
+(defun coq-parse-goals-info (goals-output)
```

Comment:
PG is a generic interface for proof assistants, see https://proofgeneral.github.io/doc/master/userman/Introducing-Proof-General/#Introducing-Proof-General, which does not only support Coq, see https://proofgeneral.github.io/doc/master/userman/Introducing-Proof-General/#Supported-proof-assistants . Assistant specific code should therefore go into the assistant specific directory.

Reply:
@Matafou Can you recomment where this code should be added in the `coq` subdir?